content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -225,6 +225,7 @@ spec:
       matchExpressions:
       - key: environment
         operator: Exists
+  validationActions: ["Deny"]
```

Comment:
NOTE: Will drop this change after https://github.com/kubernetes/website/pull/40098 is merged.